go.mod

```diff
@@ -1,13 +1,22 @@
 module github.com/open-component-model/signing-server
 
-go 1.19
+go 1.23.2
+
+replace github.com/ThalesGroup/crypto11 => github.com/mandelsoft/crypto11 v0.0.0-20241130184205-2da96e5e8173
```

Comment:
this is a fork into a private project and cannot be accepted. as we discussed we should add the relevant code in the vendored module and write an addendum in the readme instead of forking completely.